articles/azure/azs-how-add-data-managed-portal.md

```diff
@@ -0,0 +1,113 @@
+---
+title: How to add a managed disk to a virtual machine using UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides help adding managed disks to a virtual machine on UKCloud for Microsoft Azure Stack
+services: azure-stack
+author: Shaion O'Niel
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Add a managed disk to a virtual machine
+toc_sub3:
+toc_sub4:
+toc_title: Add a managed disk to a virtual machine - portal
+toc_fullpath: Users/How To/
```

Comment:
Change to:
Users/How To/Add a Managed Disk to a VM/azs-how-add-managed-disks-portal.md